vagrant/provisioning/roles/foia/templates/arkcase-portal-server-2020.16.yaml

```diff
@@ -35,3 +35,9 @@ portal:
 
 
 configServerUri: "{{ internal_host }}/config"
+
+
+loginGov: {{ loginGov_authenticated_mode | default(false) }}
```

Comment:
Is `loginGov`  a feature flag, and `Login.gov` a block with Login.gov configuration?